dist/linux/travis-utils.sh

```diff
@@ -1,7 +1,7 @@
 # This is a trimmed down copy of
 # https://github.com/travis-ci/travis-build/blob/master/lib/travis/build/templates/header.sh
 travis_time_start() {
-  travis_timer_id=$RANDOM
+  travis_timer_id=`date +%N`
```

Comment:
Unrelated change, and I fear it is wrong: the timer_id won't be uniq.

Reply:
$RANDOM exists in travis, but not inside the containers. Since I use travis-utils in build.sh, test.sh and create.sh, which are executed inside the containers, I need a replacement. Note that it is the time in nanoseconds, which is quite random, although not strictly unique. Furthermore, there not more than 10 folds to track in the same run. It'd be very bad luck if they where repeated. Last, this is only used for timing. I don't think it is a critical issue, because the run can be restarted in the worst case.

Reply:
Ok, that makes sense.  I suppose the default shell in the containers is not bash.
Worth a comment too.